build.go

```diff
@@ -176,7 +175,7 @@ func (Test) Unit() error {
 	mg.Deps(Build.BuildContainer)
 	fmt.Println("\n=====> Running Satellite Unit Tests...\n")
 	return trace.Wrap(sh.RunV(
-		"docker", "run", "-it", "--rm=true",
+		"docker", "run", "-t", "--rm",
```

Comment:
I think -t can probably be removed as well. I definitely originally wrote this from muscle memory of running docker on the cli, and I think it does cause some weird behaviour indicating there is a tty attached when there isn't actually one.